js/src/tooltip.js

```diff
@@ -16,7 +16,7 @@ const Tooltip = (($) => {
    * Check for Tether dependency
    * Tether - http://github.hubspot.com/tether/
    */
-  if (window.Tether === undefined) {
+  if (typeof Tether === 'undefined') {
```

Comment:
Why not just : ``if (Tether === undefined)`` ?
Your JSPerf prove it's faster than using ``typeof``